packages/dx-react-scheduler/docs/reference/appointments.md

```diff
@@ -56,9 +57,30 @@ Field | Type | Description
 children | ReactNode | A React node used to render the appointment content.
 data | object | An object that represents appointment data.
 
+### Appointments.SliceProps
+
+Properties passed to a component that renders a slice.
+
+Field | Type | Description
+------|------|------------
+position | 'top' &#124; 'bottom' | Specifies where appointment is sliced.
```

Comment:
What about all-day/month appts? They are sliced by right/left side. Can we use 'start' and 'end' values, or something like this.

Reply:
Renamed to `start/end`